test/OpenFeature.Contrib.Providers.Flagd.E2e.ProcessTest/Features/evaluation.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Flag evaluation
```

Comment:
I will eventually pull these in automatically with a submodule, just committing them to this branch temporarily for easy use by @bacherfl .